app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    file_name = command[1]
+    file_copy_name = command[2]
```

Comment:
it will throw an error if the `command` length is less than 3
I advise you to write it like this:
```
try:
    cp, original_file, copy_file = command.split()
except ...
```